internal/tiltfile/tiltfile_state.go

```diff
@@ -602,6 +602,22 @@ func (s *tiltfileState) assembleK8sV2() error {
 		return err
 	}
 
+	// TODO(dmiller): support all three ways of specifying a name:
+	// 1. Short name (foo) (done)
```

Comment:
The CH story defines short name is the shortest unique fragment, so it should definitely be covered by the unique fragment case, right? And that also means we can't calculate short names by calling `UniqueNames` with just tha tone entity - we need to pass them all?